frontend/src/components/common/ReleaseNotes/ReleaseNotes.tsx

```diff
@@ -49,6 +62,17 @@ export default function ReleaseNotes() {
             const storedAppVersion = helpers.getAppVersion();
             let releaseNotes = '';
             if (storedAppVersion && semver.lt(storedAppVersion as string, currentBuildAppVersion)) {
+              // attach a timeout which checks after 10 second of fetching release
+              // if the release request was not successful
+              setTimeout(() => {
+                if (
+                  !releaseDownloadURLRef.current &&
+                  !releaseFetchFailedRef.current &&
+                  !releaseNotesRef.current
+                ) {
+                  setFetchingRelease(true);
+                }
+              }, 10000);
```

Comment:
So I realize this will never happen because we have the 5 sec timeout on Octokit.
Maybe a good idea is to have this popup show up at about 3 secs after the UI is shown, so let's think 5 secs are enough. But we should then have the Octokit timeout as 10 secs. Let me know if this makes sense.

Reply:
fixed